cssgunc/CSPortal
# [Load] Added loading component and example

## Summary 

Added a loading component for when API requests are being fetched.

GIF used:

![loading-ellipses](https://user-images.githubusercontent.com/43911326/96355773-93cfed80-10b3-11eb-92a3-3e3e06672c5a.gif)

## To test 

1. Check out the loading-component branch `git checkout loading-component`
2. Run `npm start` in your command line inside the RTCPortal root directory
3. Go to the main Resources page
4. See the loading gifs before the content appears (it's really quick)

## Example

![loader2](https://user-images.githubusercontent.com/43911326/96355777-acd89e80-10b3-11eb-8e43-3de82fe9c3d0.gif)
